test/integration/net_test.go

```diff
@@ -195,6 +183,23 @@ func TestNetworkPlugins(t *testing.T) {
 	})
 }
 
+// verifyHairpinMode makes sure the hairpinning (https://en.wikipedia.org/wiki/Hairpinning) is correctly configured for given CNI
+func verifyHairpinMode(t *testing.T, ctx context.Context, profile string, hairpin bool) {
```

Comment:
why this didnt get added to the Integration Test Docs ?
https://minikube.sigs.k8s.io/docs/contrib/tests/

maybbe the `make generate-docs` expects it to be called validateHairPinMode...

Reply:
yeah, `generate-docs` looks only for `validate*` functions. fixed